### ISSUE TITLE
Can't add books to a buy shop

### ISSUE THREAD

**Issue by crashtheparty** [Majncraft/BetterShops#5]
16.09 21:36:12 [Server] WARN [BetterShops] Plugin BetterShops v2.2.3-34 generated an exception while executing task 2071
16.09 21:36:12 [Server] INFO java.lang.NullPointerException: null
16.09 21:36:12 [Server] INFO at max.hubbard.bettershops.Listeners.AddItemListener$1.run(AddItemListener.java:97) ~[?:?]
16.09 21:36:12 [Server] INFO at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71) ~[spigot-1.12.1_9_14.jar:git-Spigot-da42974-8f47214]
16.09 21:36:12 [Server] INFO at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52) [spigot-1.12.1_9_14.jar:git-Spigot-da42974-8f47214]
16.09 21:36:12 [Server] INFO at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:1.8.0_112]
16.09 21:36:12 [Server] INFO at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:1.8.0_112]
16.09 21:36:12 [Server] INFO at java.lang.Thread.run(Thread.java:745) [?:1.8.0_112]

**Comment by dwi** [comment]
Enchanting books? Empty books? Written books? Adding them to a buy or sell shop? Adding them as a new item or filling up a stock?

Be a little more specific

**Comment by crashtheparty** [comment]
It was upon trying to add a regular book (Material.BOOK) as a new item to the buy shop of the shop (I tried to add them to more than one). Adding them to the sell shop worked just fine. I did not try any other kinds of books.

And actually, I just went back and tested it some more, and it doesn't look like I can add ANYTHING to the buy shop. It just keeps generating the exception above on me when I try to add anything. Downloaded the latest Jenkins build just to make sure and nope it doesn't work at all.
